docs/tutorials/thresholding.md

```diff
@@ -34,7 +34,7 @@ A *thresholder* in QuPath is something that can apply a threshold to an image wh
 
 ### Thresholds in action
 
-You can create a thresholder using the *Create thresholder* command.
+You can create a thresholder using the {menuselection}Classify --> Pixel classification --> Create thresholder> command.
```

Comment:
This doesn't display properly, it needs to be
```
{menuselection}`Classify --> Pixel classification --> Create thresholder`
```

Reply:
section amended to 
`{menuselection}`Classify --> Pixel classification --> Create thresholder``